drivers/sensor/apds9960/apds9960.c

```diff
@@ -28,7 +28,7 @@ LOG_MODULE_REGISTER(APDS9960, CONFIG_SENSOR_LOG_LEVEL);
 
 static void apds9960_handle_cb(struct apds9960_data *drv_data)
 {
-	apds9960_setup_int(drv_data, false);
+	apds9960_setup_int(drv_data->dev->config, false);
```

Comment:
note: does this work without de-referencing first to config type? I actually see that drv_data holds a config reference (probably for the work queue context).

Reply:
I removed the `drv_data` config reference.  The compiler seems fine with it, guessing that there is an implicit cast between the `const void *` and `const struct apds9960_config *`